cmake/Modules/CodeCoverage.cmake

```diff
@@ -283,7 +301,7 @@ function(SETUP_TARGET_FOR_COVERAGE_GCOVR_HTML)
         COMMAND ${CMAKE_COMMAND} -E make_directory ${PROJECT_BINARY_DIR}/${Coverage_NAME}
 
         # Running gcovr
-        COMMAND ${Python_EXECUTABLE} ${GCOVR_PATH} --html --html-details
+        COMMAND ${GCOVR_PATH} --html --html-details
```

Comment:
They did this for the XML report, but not for the HTML report. GCovr won't run without it, if the Python executable the CMake build picks up isn't the **correct** one to run `gcovr` under, as it _isn't_ on my system. But just executing the command the way it's intended to be run works fine.

In some ways I like the `gcovr` report better, as it's flatter (I unexpectedly found that to be more conducive to exploring the results), and it includes branch coverage where `lcov` didn't even when I force-enabled it in the command-line args. (Branches were added to the report, but there was no data.) The output of `gcovr` is definitely a little less pretty and a little more 'raw' than lcov's, though.